infrastructure/kubernetes/spid-testenv2.yml

```diff
@@ -0,0 +1,307 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spid-testenv2-configmap
+  namespace: default
+data:
+  config.yml: |
+    #
+    # Identity provider configuration
+    #
+    base_url: "http://localhost:8088"
```

Comment:
shouldn't this be the public URL (I guess it's used for generating links)?

Reply:
I guess this is the backend URL (python web server) while the public one is the one from the ingress (but I'm not sure, @alexrj  ?)